src/editor/Core/Theme.re

```diff
@@ -22,6 +22,20 @@ module EditorColors = {
     editorMenuForeground: Color.t,
     editorMenuItemSelected: Color.t,
     tabActiveForeground: Color.t,
+    statusBarItemVisualModeBackground: Color.t,
+    statusBarItemInsertModeBackground: Color.t,
+    statusBarItemReplaceModeBackground: Color.t,
+    statusBarItemNormalModeBackground: Color.t,
+    statusBarItemOperatorModeBackground: Color.t,
+    statusBarItemCommandlineModeBackground: Color.t,
+    statusBarItemVisualModeForeground: Color.t,
+    statusBarItemInsertModeForeground: Color.t,
+    statusBarItemReplaceModeForeground: Color.t,
+    statusBarItemNormalModeForeground: Color.t,
+    statusBarItemOperatorModeForeground: Color.t,
+    statusBarItemCommandlineModeForeground: Color.t,
+    statusBarItemForeground: Color.t,
```

Comment:
Should we just use `statusBar.background` and `statusBar.foreground` for these?

Thinking of how they relate here:
https://code.visualstudio.com/api/references/theme-color#status-bar-colors